1738262928.075945 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:98 __init__ INFO: Starting DataFlowKernel with config
Config(
    app_cache=True, 
    checkpoint_files=None, 
    checkpoint_mode=None, 
    checkpoint_period=None, 
    dependency_resolver=None, 
    executors=(ThreadPoolExecutor(
        label='threads', 
        max_threads=1, 
        storage_access=None, 
        thread_name_prefix='', 
        working_dir=None
    ),), 
    exit_mode='cleanup', 
    garbage_collect=True, 
    initialize_logging=True, 
    internal_tasks_max_threads=1, 
    max_idletime=120.0, 
    monitoring=None, 
    retries=0, 
    retry_handler=None, 
    run_dir='runinfo', 
    std_autopath=None, 
    strategy='simple', 
    strategy_period=5, 
    usage_tracking=False
)
1738262928.076055 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:100 __init__ INFO: Parsl version: 2024.07.29
1738262928.076397 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.usage_tracking.usage:118 __init__ DEBUG: Tracking level: False
1738262928.076606 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:124 __init__ INFO: Run id is: d87a3f14-7827-4447-9467-12c1f762355d
1738262928.105667 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:131 __init__ DEBUG: Considering candidate for workflow name: /home/beams/PTYCHOSAXS/miniconda3/envs/midas/lib/python3.12/site-packages/parsl/dataflow/dflow.py
1738262928.105720 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:131 __init__ DEBUG: Considering candidate for workflow name: /home/beams/PTYCHOSAXS/miniconda3/envs/midas/lib/python3.12/site-packages/parsl/dataflow/dflow.py
1738262928.105744 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:131 __init__ DEBUG: Considering candidate for workflow name: /home/beams/PTYCHOSAXS/opt/MIDAS/FF_HEDM/v7/ff_MIDAS.py
1738262928.106018 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:137 __init__ DEBUG: Using ff_MIDAS.py as workflow name
1738262928.106087 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.memoization:163 __init__ INFO: App caching initialized
1738262928.106117 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.jobs.strategy:145 __init__ DEBUG: Scaling strategy: simple
1738262928.108362 2025-01-30 12:48:48 MainProcess-586366 MainThread-139765920188224 parsl.dataflow.dflow:1202 atexit_cleanup WARNING: Python is exiting with a DFK still running. You should call parsl.dfk().cleanup() before exiting to release any resources
